ipinfo/detailed_help.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import "fmt"
+
+var DetailedHelp = fmt.Sprintf(`Usage: %s <cmd> [<opts>] [<args>]
```

Comment:
As we add more help messages for more subcommands, ideally it's clear which command it's for. So can we move this below the short-help in `ipinfo/cmd_default.go` and use camel case for a lower case first char?